<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandLinesClass</class>
 <widget class="QWidget" name="CommandLinesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CommandLines</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Commands</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidgetWithMenu" name="cmd_list">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_btn">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Uboard_NMEA">
         <property name="text">
          <string>Uboard_NMEA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="HSD_PORTU">
         <property name="text">
          <string>HSD_PORTU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="HSD_PORTI">
         <property name="text">
          <string>HSD_PORTI</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>History</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidgetWithMenu" name="history_list">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_btn">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QListWidgetWithMenu</class>
   <extends>QListWidget</extends>
   <header>custom/QListWidgetWithMenu.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
